privacyidea/lib/eventhandler/counterhandler.py

```diff
@@ -44,7 +44,7 @@ class CounterEventHandler(BaseEventHandler):
     """
 
     identifier = "Counter"
-    description = "This event handler increases arbitrary counters in the database."
+    description = "This event handler increases, decreases or resets arbitrary counters in the database."
```

Comment:
FYI: This is correct but has nothing to do with the fail counter.

Reply:
that's true. It is the change in https://github.com/privacyidea/privacyidea/pull/2490/commits/b1e02548ac4586789dc36bb2d07e9fe79f926b11